src/lib/timing.h

```diff
@@ -24,22 +24,35 @@
 #define NANOS_IN_MILLISEC 1000000
 #define NANOS_IN_SEC      1000000000
 
+#if PHP_WIN32 && (__i386__ || __x86_64__ || _M_IX86 || _M_X64)
+#define WIN_SUPPORTS_RDTSC 1
+#else
+#define WIN_SUPPORTS_RDTSC 0
+#endif
```

Comment:
This is also from the other PR (#629), they need to be untangled.

Reply:
yes, #629 must be finished first